installer.tpl.sh

```diff
@@ -89,7 +82,33 @@ detect_os ()
   echo "Detected operating system as ${os}/${dist}."
 }
 
-detect_ver ()
+setup_type ()
+{
+  if [ "${repo_type}" = "release" ]; then
+    if [ "${ver}" = "2" ]; then
+      repo_path="release/series-"
+    else
+      repo_path="release/"
+    fi
+  elif [ "${repo_type}" = "pre-release" ]; then
+    if [ "${ver}" = "2" ]; then
+      repo_path="pre-release/series-"
+    else
+      echo "pre-release repository can be set only with Tarantool series-2"
```

Comment:
How about giving a link to the [release policy document](https://github.com/tarantool/tarantool/discussions/6182) here (and for 'live' in series-2 below)?